test/integration/create_account_test.rb

```diff
@@ -4,7 +4,7 @@
 #  Cryptopus and licensed under the Affero General Public License version 3 or later.
 #  See the COPYING file at the top-level directory or at
 #  https://github.com/puzzle/cryptopus.
-
+require 'pry'
```

Comment:
pry ... never check in pry ..

Reply:
require 'pry'; binding.pry 
you can use it like this as a one-liner. no need to add it on top of a file